manifests/k8s.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: go-todo-list
```

Comment:
let's rename `go-todo-list` -> `todo-list` everywhere. the programming language isn't important here